packages/client/src/components/Conclusion/Conclusion.tsx

```diff
@@ -1,65 +1,137 @@
 import styled from "styled-components";
 
-import { sections } from "../../config/matomo";
+import { Topic, urls } from "../../config";
+import { eventNames, sections } from "../../config/matomo";
 import withTracking, { MatomoTrackEventProps } from "../../hoc/withTracking";
 import Disclaimer from "../Disclaimer";
 import Markdown from "../Markdown";
-import { ConclusionOutcome, NeedPermitContent, NoPermitDescription } from ".";
+import ConclusionOutcome from "./ConclusionOutcome";
+import {
+  DemolitionNeedReport,
+  DemolitionPermitFree,
+  NeedPermit,
+  PermitFree,
+} from "./content";
 
 const ConclusionWrapper = styled.div`
   @media screen {
     padding-bottom: ${themeSpacing(5)};
   }
 `;
-const { NEED_CONTACT, NEED_PERMIT, PERMIT_FREE } = imtrOutcomes;
 
-const Conclusion: React.FC<{ checker: Checker } & MatomoTrackEventProps> = ({
+const {
+  NEED_BOTH_PERMIT_AND_REPORT,
+  NEED_CONTACT,
+  NEED_PERMIT,
+  NEED_REPORT,
+  PERMIT_FREE,
+} = clientOutcomes;
+
+type ConclusionProps = {
+  checker: Checker;
+  topic: Topic;
+};
+
+const Conclusion: React.FC<ConclusionProps & MatomoTrackEventProps> = ({
   checker,
   matomoTrackEvent,
+  topic,
 }) => {
   const { t } = useTranslation();
 
   // Get all the outcomes to display
   const outcomes = checker.getOutcomesToDisplay();
-  const outcomeType = checker.outcomeType();
 
   // The "need contact" content comes from the IMTR file itself
+  const outcomeType = checker.getClientOutcomeType();
+  // This function can be moved to `imtr-client`
   const getNeedContactContent = outcomes.find(
     ({ outcome }: { outcome: string }) => outcome === imtrOutcomes.NEED_CONTACT
   );
 
   // Define the content for the Conclusion components
   const contents = {
-    [NEED_CONTACT]: {
-      mainContent: (
-        <Markdown
-          eventLocation={sections.CONCLUSION}
-          source={getNeedContactContent?.description || ""}
-        />
-      ),
-      title: getNeedContactContent?.title || "",
-    },
-    [NEED_PERMIT]: {
-      mainContent: <NeedPermitContent />,
-      title: t("outcome.needPermit.you need a permit"),
+    // This is the default content
+    default: {
+      [NEED_CONTACT]: {
+        mainContent: (
+          <Markdown
+            eventLocation={sections.CONCLUSION}
+            source={getNeedContactContent?.description || ""}
+          />
+        ),
+        title: getNeedContactContent?.title || "",
+      },
+      [NEED_PERMIT]: {
+        mainContent: <NeedPermit />,
+        title: t("outcome.needPermit.you need a permit"),
+      },
+      [PERMIT_FREE]: {
+        footerContent: <PermitFree />,
+        title: t("outcome.permitFree.you dont need a permit"),
+      },
     },
-    [PERMIT_FREE]: {
-      footerContent: <NoPermitDescription />,
-      title: t("outcome.permitFree.you dont need a permit"),
+    // This content is only relevant for the demolition checker
+    demolition: {
+      [NEED_BOTH_PERMIT_AND_REPORT]: {
```

Comment:
Good idea! But I can already see a lot of merge conflicts coming with the custom-hooks PR, because there's also some changes in text in that PR. Better to write this down and place on the backlog. What do you think?

Reply:
Marked this as a separate Todo